tests/test_algorithms.py

```diff
@@ -83,3 +83,51 @@ def test__split_tests_calculates_avg_test_duration_only_on_present_tests(self, a
         expected_first, expected_second = expected
         assert first.selected == expected_first
         assert second.selected == expected_second
+
+    def test_each_group_is_assigned_a_test_front_loaded(self):
+        from collections import namedtuple
+
+        from pytest_split import algorithms
+
+        item = namedtuple("item", "nodeid")
+
+        durations = {}
+        durations["a"] = 100
+        durations["b"] = 1
+        durations["c"] = 1
+        durations["d"] = 1
+        durations["e"] = 1
+        durations["f"] = 1
+        durations["g"] = 1
+        durations["h"] = 1
+
+        items = [item(x) for x in ["a", "b", "c", "d", "e", "f", "g", "h"]]
```

Comment:
Would it be the same if the loop would be over `sorted(durations.keys())`?

Reply:
What do you mean, sorry?

Reply:
I think he's saying you can do `items = sorted(durations.keys())` instead of duplicating all of the items.

Reply:
yeah I'm basically asking if this would be the same:
```python
items = [item(x) for x in sorted(durations.keys())]
```

IMO it would be better considering readability as then it's explicit that there's one item for each thingy in `durations`

Reply:
I just copied in the example test from [here](https://github.com/jerry-git/pytest-split/issues/26#issuecomment-865649265) and made sure we had one example that was extremely front-loaded and one that was extremely back-loaded. I can make it less explicit if you prefer.